assets/prometheus-k8s/rules.yaml

```diff
@@ -927,8 +927,7 @@ spec:
       record: cluster:kubelet_volume_stats_used_bytes:provisioner:sum
     - expr: sum(etcd_object_counts) BY (instance)
       record: instance:etcd_object_counts:sum
-    - expr: max(etcd_object_counts{resource=~"builds\\.build\\.openshift\\.io|deploymentconfigs\\.apps\\.openshift\\.io|images\\.image\\.openshift\\.io|statefulsets\\.apps|deployments\\.apps|cronjobs\\.batch|jobs\\.batch|pods|persistentvolumeclaims|persistentvolumes|services|namespaces|routes\\.route\\.openshift\\.io|ingresses\\.networking\\.k8s\\.io|horizontalpodautoscalers\\.autoscaling|users\\.user\\.openshift\\.io|machinesets\\.machine\\.openshift\\.io|resourcequotas"})
-        by (resource)
+    - expr: max(etcd_object_counts) by (resource)
```

Comment:
Seems like with this change we edging close to the limit we set in our origin test (its at 473 series we sent from this CI run), do you mind adjusting that limit to a reasonable one as well, so we don't get flaky or failing tests. Thanks!
https://github.com/openshift/origin/blob/712d7c621fe4353077f889bb754fb55827dbabcf/test/extended/prometheus/prometheus.go#L103-L106

Reply:
https://github.com/openshift/origin/pull/25592